src/msix/AppxBundleManifest.cpp

```diff
@@ -187,14 +192,22 @@ namespace MSIX {
     HRESULT STDMETHODCALLTYPE AppxBundleManifestPackageInfo::GetResources(IAppxManifestQualifiedResourcesEnumerator **resources) noexcept try
     {
         ThrowErrorIf(Error::InvalidParameter, (resources == nullptr || *resources != nullptr), "bad pointer.");
-        std::vector<ComPtr<IAppxManifestQualifiedResource>> languages;
+        std::vector<ComPtr<IAppxManifestQualifiedResource>> m_resources;
         for(auto& bcp47 : m_languages)
         {
             auto resource = ComPtr<IAppxManifestQualifiedResource>::Make<AppxBundleQualifiedResource>(m_factory, bcp47.GetFullTag());
-            languages.push_back(std::move(resource));
+            m_resources.push_back(std::move(resource));
         }
+
+        for (auto& scale : m_scales)
+        {
+	        std::string language;
+	        auto resource = ComPtr<IAppxManifestQualifiedResource>::Make<AppxBundleQualifiedResource>(m_factory, language, scale);

```

Comment:
>Make<AppxBundleQualifiedResource>(m_factory, language, scale); [](start = 65, length = 62)

You don't need the empty string. Create a constructor that takes the factory and the scale.